docs/MoreAboutPersistentStorage.md

```diff
@@ -1,31 +1,29 @@
-| thanos-receive-default | Thanos receiver accepts incoming data (Prometheus remote-write requests) and writes these into a local instance of the Prometheus TSDB. Periodically (every 2 hours), TSDB blocks are uploaded to the object storage for long term storage and compaction. The amount of hours or days of data retained in this stateful set, which acts a local cache was fixed in API Version `observability.open-cluster-management.io/v1beta`. It has been exposed as an API parameter in `observability.open-cluster-management.io/v1beta2`: _RetentionInLocal_ |
+| thanos-rule |The thanos ruler evaluates Prometheus recording and alerting rules against a chosen query API by issuing queries at a fixed interval. Rule results are written back to the disk in the Prometheus 2.0 storage format. Rule results are written back to disk in the Prometheus 2.0 storage format. The amount of hours or days of data retained in this stateful set was fixed in the API version, `observability.open-cluster-management.io/v1beta1`. It has been exposed as an API parameter in `observability.open-cluster-management.io/v1beta2`: `_RetentionInLocal_` |
+| thanos-receive-default | Thanos receiver accepts incoming data (Prometheus remote-write requests) and writes these into a local instance of the Prometheus TSDB. Periodically (every 2 hours), TSDB blocks are uploaded to the object storage for long term storage and compaction. The amount of hours or days of data retained in this stateful set, which acts a local cache was fixed in API Version `observability.open-cluster-management.io/v1beta`. It has been exposed as an API parameter in `observability.open-cluster-management.io/v1beta2`: `_RetentionInLocal_` |
 | thanos-store-shard| It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access. It keeps a small amount of information about all remote blocks on local disk and keeps it in sync with the bucket. This data is generally safe to delete across restarts at the cost of increased startup times. |
 
 
+### Configuring the stateful sets
 
-
-
-### Configuring the Stateful sets
-
-In Mutlicluster Observability Operator API: `observability.open-cluster-management.io/v1beta1` following was exposed:
+The following information is exposed in the Mutlicluster Observability Operator API, `observability.open-cluster-management.io/v1beta1`:
 
 ```
     //defaults shown below
     statefulSetSize: 10Gi
     statefulSetStorageClass: gp2
 ```
-Taking into consideration this can result in wasted space, we have allowed settings to be individually tunable in the next revision:  `observability.open-cluster-management.io/v1beta2`
+
+Considering the information from the API, this can result in wasted space. You can now update the following parameters individually in `observability.open-cluster-management.io/v1beta2`. View the following parameters that can be updated:
```

Comment:
Statement agreed upon with @bjoydeep 

In the `observability.open-cluster-management.io/v1beta1` API, one fixed size is used for all of the stateful sets, which can result in wasted space. You can now update the stateful sets individually in the `observability.open-cluster-management.io/v1beta2` API. As shown in the following example: